src/Chunking.cpp

```diff
@@ -105,13 +215,28 @@ void Chunking::addEntityToChunks(std::set<Chunk*> chnks, const EntityRef ref) {
                 ((Mob*)other)->playersInView++;
         }
     }
+
+    if (ref.kind != EntityKind::PLAYER)
+        return; // nothing to send
```

Comment:
nit: While I normally advocate for short-circuit style, in this case I think it's better to enclose the code below in an `if` block to match the one above. It makes more sense conceptually, as it's a special case where the `ref` is a `Player`, and it's safer in the hypothetical scenario where we add more general logic after that block.